client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/configure-sms-notification/index.tsx

```diff
@@ -1,25 +1,35 @@
 import { Button } from '@automattic/components';
 import { Icon, plus } from '@wordpress/icons';
 import { useTranslate } from 'i18n-calypso';
+import { StateMonitorSettingsSMS } from '../../sites-overview/types';
+import SMSItemContent from './sms-item-content';
+
+import './style.scss';
```

Comment:
It looks like this file doesn't exist anymore. 

Unable to run the build -

<img width="1467" alt="Screenshot 2023-06-09 at 11 53 26 AM" src="https://github.com/Automattic/wp-calypso/assets/10586875/5c401216-2f02-42b4-b98c-e56d0ee9374a">